commands/certmanager/get.go

```diff
@@ -0,0 +1,72 @@
+package certmanager
+
+import (
+	"context"
+
+	"github.com/ionos-cloud/ionosctl/pkg/core"
+	"github.com/spf13/cobra"
+
+	sdkgo "github.com/ionos-cloud/sdk-go-cert-manager"
+)
+
+func CertGetCmd() *core.Command {
+	cmd := core.NewCommand(context.TODO(), nil, core.CommandBuilder{
+		Namespace:  "certmanager",
+		Resource:   "certificates",
+		Verb:       "get",
+		Aliases:    []string{"g"},
+		ShortDesc:  "Get Certificate by ID",
+		LongDesc:   "Use this command to retrieve a Certificate by ID.",
+		Example:    "ionosctl certificate-manager get --certificate-id 47c5d9cc-b613-4b76-b0cc-dc531787a422",
+		PreCmdRun:  PreCmdGet,
+		CmdRun:     CmdGet,
+		InitClient: true,
+	})
+
+	cmd.AddStringFlag(CertId, "i", "", "Response get a single certificate (required)")
```

Comment:
Don't hard-code `(required)` into the flag description. Instead, add `core.RequiredFlagOption()` as an argument to `AddStringFlag`:

`cmd.AddStringFlag(CertId, "i", "", "Response get a single certificate", core.RequiredFlagOption())`

Reason: Was hoping we could automate pre-runs someday, to just use these `core.Options` funcs, even though they are legacy code they seem to be a good idea, at least in theory. One could argue here YAGNI but there's only so much we can refactor at once.